docs/index.html

```diff
@@ -181,14 +183,10 @@
 evaluates to the `integer` `3`.  The `{}` block ensures that `one` and `two`
 are not accessible after `three` is computed.
 
-```reason
-let three = {
-  let one = 1;
-  let two = 2;
-  one + two
-};
-/* one and two not accessible here! */
-```
+<pre>
+<p data-id="-P8x86Mg6w" data-host="replay-154206.appspot.com" data-height="256" data-font-size="12px" class="replay" ></p>
```

Comment:
This api call has only a hash value. Is it possible to preserve the original code as part of the api call so it is more maintainable? 

As an example, see Golang's plugin code:
![image](https://cloud.githubusercontent.com/assets/1017689/21927903/bee683c4-d93c-11e6-8d7d-765cd33bdf84.png)